src/Mod/PartDesign/Gui/TaskRevolutionParameters.cpp

```diff
@@ -176,23 +179,23 @@ void TaskRevolutionParameters::addAxisToCombo(App::DocumentObject* linkObj,
                                               QString itemText)
 {
     this->ui->axis->addItem(itemText);
-    this->axesInList.push_back(new App::PropertyLinkSub());
+    this->axesInList.push_back(std::move(std::make_unique<App::PropertyLinkSub>()));

```

Comment:
Write `this->axesInList.emplace_back();`

Reply:
I realized a problem with the above hint. It should be `this->axesInList.emplace_back(new App::PropertyLinkSub());`because now it creates a unique pointer with null. This will result into a crash. I will fix it.